server/src/http/routes/appointmentRequest.controller.ts

```diff
@@ -173,6 +179,35 @@ export default ({ users, appointments, mailer, widgetParameters, etablissements
     })
   )
 
+  router.post(
+    "/reply",
+    tryCatch(async (req, res) => {
+      await appointmentReplySchema.validateAsync(req.body, { abortEarly: false })
+      const paramsAppointementItem = req.body
```

Comment:
```suggestion
      const paramsAppointementItem = req.body
```
You can destructure like 

```
const { appointmentId ...} = req.body
```

Reply:
@bdavidxyz à faire